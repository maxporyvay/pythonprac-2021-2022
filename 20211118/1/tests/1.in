class Str(str):
	def __del__(self):
		pass
@objcount
class MyStr(Str):
	def __del__(self):
		print('deleted')
		super().__del__()
a = MyStr('a')
b = MyStr('b')
print(MyStr.counter)
del a
del b
print(MyStr.counter)