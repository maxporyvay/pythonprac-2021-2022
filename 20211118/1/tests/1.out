2
deleted
deleted
0
